docs/sources/setup/upgrade/upgrade-to-6x/index.md

```diff
@@ -124,6 +124,57 @@ In Helm chart version 6.34.0, [PR #18558](https://github.com/grafana/loki/pull/1
 **Why this change was necessary**: 
 The previous configuration caused ingester scaling operations to fail because the rollout-operator couldn't find the correct headless services for the `/ingester/prepare-downscale` endpoint.
 
+#### BREAKING: Make access modes for persistence on all PVCs and StatefulSets editable (6.38.0+)
+
+This PR introduces the ability to edit the access modes for persistence on all PVCs and StatefulSets. This is a breaking change because it requires users to manually orphan StatefulSets before upgrading.
+
+**Required action before upgrading to 6.38.0**:
+
+**Option 1: Use the automated upgrade script (recommended)**:
+
+Download and run the upgrade script that will automatically detect and orphan only the StatefulSets that exist in your deployment:
+
+```bash
+# Download the upgrade script
+curl -O https://raw.githubusercontent.com/grafana/loki/main/production/helm/loki/scripts/upgrade-to-6.38.0.sh
+chmod +x upgrade-to-6.38.0.sh
+
+# Run the upgrade script
+./upgrade-to-6.38.0.sh -r <release-name> [-n <namespace>] [-v <target-version>]
```

Comment:
Unclear what I should be providing here for release name?  If it's my current Loki version, then maybe we should change this to `<current-version>` to clarify?